<?xml version="1.0" encoding="utf-8"?>
<launch>
  <!-- Param -->
  <rosparam file="$(find happymimi_params)/param/object_mapping.yaml" command="load" ns="/object_mapping" />
  <rosparam file="$(find actplan_executor)/config/act_state.yaml" command="load" ns="/act_state" />

  <!-- Node -->
  <node name="exe_actplan" pkg="actplan_executor" type="exe_actplan.py" output="screen" />
  <!-- <node name="test" pkg="actplan_executor" type="test.py" /> -->

</launch>
